client/src/Components/CreateOffer/index.jsx

```diff
@@ -1,11 +1,186 @@
-import React from 'react';
-import UnderConstruction from '../UnderConstruction';
-
-export default function AutoCompleteTags() {
-  return (
-    <>
-      <h1>AutoCompleteTags</h1>
-      <UnderConstruction />
-    </>
-  );
+import React, { Component } from 'react';
+import { Container, Row, Col, Form, Button } from 'react-bootstrap';
+import { withRouter } from 'react-router-dom';
+
+import AutoCompleteTags from '../CommonComponents/AutoCompleteTags';
+import newOfferValidation from './validationSchema';
+
+import './style.css';
+
+class CreateOffer extends Component {
+  state = {
+    title: '',
+    position: '',
+    description: '',
+    skills: null, // skills id's
+    offerType: null, // offer_types id's
+    // member_id: '',
+    errMsg: '',
+  };
+
+  componentDidMount() {
+    // const member_id = get member_id from local storage
+    // this.setState({ member_id })
+  }
+
+  handleSubmit = () => {
+    // eslint-disable-next-line react/prop-types
+    // const { history } = this.props;
+    const { title, position, description, offerType } = this.state;
+
+    newOfferValidation
+      .validate(
+        { title, position, description, offerType },
+        { abortEarly: false }
+      )
+      .then(() => {
+        this.setState({ errMsg: '' });
+        // send request to the backend with body
+        // axios
+        // .post('/api/v1/offers', { title, position, description, skills, offerType })
+        // get offer_id from response
+        // history.push(`/app/${offer_id}`);
+        // send request with new skills from memeber (which don't exist in autocomplete) to update filters table `PUT` : `/api/v1/filter/:memberId`
+        // and the same for offerTypes
+        // body of the request { skills: this.state.skills , offerTypes: this.state.offerTypes}
+      })
+      .catch(({ inner }) => {
+        if (inner) {
+          const errors = inner.reduce(
+            (acc, item) => ({ ...acc, [item.path]: item.message }),
+            {}
+          );
+          this.setState({ errMsg: { ...errors } });
+        }
+      });
+  };
+
+  handleTitle = title => this.setState({ title });
+
+  handlePosition = position => this.setState({ position });
+
+  handleDescription = description => this.setState({ description });
```

Comment:
group handleTitle, handlePosition, handleDescription in one function, something like:
```js
handleInput = event => {
  const { name, value } = event.target;
  this.setState({ [name]: value });
};
```
also you'd need to change it on the component tag soemthing like this
```jsx
<Form.Control
  type="text"
  name="title"
  placeholder="eg:  Small Shop seeks workers "
  onChange={this.handleInput}
 />
```